tensorflow/src/main/scala/magnolify/tensorflow/ExampleType.scala

```diff
@@ -100,6 +118,36 @@ object ExampleField {
         p.typeclass.put(f, key(k, cm.map(p.label)), p.dereference(v))(cm)
         f
       }
+
+    override protected def schemaInternal(k: String)(cm: CaseMapper): Schema = {
+      val sb = Schema
+        .newBuilder()
+        .setAnnotation(getDoc(caseClass.annotations, caseClass.typeName.full))
+      caseClass.parameters.foldLeft(sb) { (b, p) =>
+        val s = p.typeclass.schemaInternal(key(k, cm.map(p.label)))(cm)
+        val annotatedFs = s.getFeatureList.asScala
+          .map { f =>
+            val annotation = if (f.hasAnnotation) {
+              f.getAnnotation
+            } else {
+              getDoc(p.annotations, s"${caseClass.typeName.full}#${key(k, cm.map(p.label))}")
```

Comment:
just wondering, what's the rationale for this behavior (as opposed to just not calling `.setAnnotation` if `!f.hasAnnotation`)?

Reply:
not sure I understand the question. We need the getDoc because the primitive example types do not add the annotations to their schema (because the schema is on the record and not on the field?)